Remotion/ObjectBinding/Core/IBusinessObjectEnumerationProperty.cs

```diff
@@ -49,7 +49,7 @@ public interface IBusinessObjectEnumerationProperty: IBusinessObjectProperty
     /// The <see cref="IEnumerationValueInfo"/> object for the provided <paramref name="value"/> or <see langword="null"/> if the 
     /// <paramref name="value"/> represents <see langword="null"/>. 
     /// </returns>
-    IEnumerationValueInfo GetValueInfoByValue (object value, IBusinessObject businessObject);
+    IEnumerationValueInfo? GetValueInfoByValue (object? value, IBusinessObject? businessObject);
```

Comment:
Please recheck this. I think that the nullability of the IBusinessObject parameter should be consistent on all four methods on the interface.